crates/rpc/rpc/src/layers/jwt_secret.rs

```diff
@@ -26,6 +26,8 @@ pub enum JwtError {
     MissingOrInvalidAuthorizationHeader,
     #[error("JWT decoding error {0}")]
     JwtDecodingError(String),
+    #[error("IO error occurred while reading {path}: {err}")]
```

Comment:
"reading" is probably a sufficient context that we interpreted this as a file, I think